packages/create-sitecore-jss/src/templates/angular/webpack.config.js

```diff
@@ -1,7 +1,11 @@
-const Dotenv = require("dotenv-webpack");
+const Dotenv = require('dotenv-webpack');
+const path = require('path');
 
 module.exports = {
-  plugins: [
-    new Dotenv(),
-  ],
-}
+  plugins: [new Dotenv()],
+  resolve: {
+    alias: {
+      '@sitecore-cloudsdk': path.resolve(__dirname, 'node_modules/@sitecore-cloudsdk'),
```

Comment:
is there a way to only apply this on monorepo?

Reply:
I think so but then we might need to setup a global or maybe shared webpack configuration which is out of scope for this task.
Worth exploring when we setup Nx.

Reply:
But how? it's just an alias which is telling webpack how to resolve module paths during bundling. It's not changing how dependencies are installed or managed in node_modules. Dependency hoisting is concept associated with package managers.

Reply:
The scenario I have in mind is this: 
1. `@sitecore/cloudsdk` installed outside of angular folder
2. webpack cannot resolve `node_modules/@sitecore-cloudsdk` from within angular folder because of this
3. Error?

This is my last concern, I'll approve the PR, but please only merge after you verified this webpack configuration works with our pnpm setup (angular builds, runs in dev and prod modes).

Reply:
`@sitecore/cloudSdk` is installed within the angular app and not outside.
This webpack configuration just makes sure that in case of multiple `@sitecore/cloudsdk` instances it gets resolved to one.